### PR TITLE
Feature/Improve Bottom Menu for Multi chapter Publications

### PR DESCRIPTION
**Pull request by ThatcherK**
ticket #1079

### CODE REVIEW

File: di_website/publications/models.py [hunk 1]
```diff
@@ -431,7 +431,7 @@ def foreword(self):
 
     @cached_property
     def summary(self):
-        return get_first_child_of_type(self, PublicationSummaryPage)
+        return get_first_child_of_type (self, PublicationSummaryPage)
```
Comment (edwinmp) [inline di_website/publications/models.py:434]: Remove space

File: di_website/publications/templatetags/report_utils.py [hunk 2]
```diff
@@ -2,32 +2,27 @@
...
-    if not chapters and not chapter_number:
-        return ''
+def get_next_page(all_pages, label):
     try:
-        chapters = list(chapters)
-        for index, item in enumerate(chapters):
-            if int(item.chapter_number) == int(chapter_number):
-                if index < len(chapters) - 1:
-                    return chapters[index + 1]
+        print(all_pages)
```
Comment (edwinmp) [inline di_website/publications/templatetags/report_utils.py:20]: Remove

**Review by edwinmp — CHANGES_REQUESTED** [review]
Excellent work on this.

On the initial report page, there's a block that should not be there on the `Jump to Section` - see screenshot

image

Lastly, this change doesn't seem to have been applied to Short Publication pages

**Comment by ThatcherK** [comment]
@edwinmp originally, this prev-next menu was used for multi-chapter publications, which does not apply to short publications. wondering how to make it work for them, seeing as its simply a single page, with the 'jump to section' menu, that lets you hope to the different sections in the publication. Thoughts?

**Comment by edwinmp** [comment]
> 

You're right ... The Jump To Section widget in Short Publication pages is for internal navigation and doesn't need this update.

**Review by edwinmp — DISMISSED** [review]

**Comment by ThatcherK** [comment]
Then I guess my work here is done

**Review by davidebukali — DISMISSED** [review]

**Review by davidebukali — APPROVED** [review]

**Pull request merged**